src/pages/CompareCountries.js

```diff
@@ -13,18 +13,24 @@ function CompareCountries(props) {
     let [chartOptions, setChartOptions] = useState(undefined);
```

Comment:
Looks good, Matt. Could we also add some sort of alert for when users hit the compare button, but there are not countries selected? Additionally, maybe don't show the chart if there are no countries selected - I can see the title appearing.